assets/js/components/adminbar/AdminBarZeroData.js

```diff
@@ -19,15 +19,19 @@
 /**
  * WordPress dependencies
  */
-import { withFilters } from '@wordpress/components';
-import { Component, Fragment } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
 
-class LegacyAdminBarModules extends Component {
-	render() {
-		return (
-			<Fragment />
-		);
-	}
-}
+const AdminBarZeroData = () => {
+	return (
+		<div>
+			<h4>
+				{ __( 'No data available yet', 'google-site-kit' ) }
+			</h4>
+			<p>
+				{ __( 'There is no data available for this content yet. This could be because it was recently created or because nobody has accessed it so far.', 'google-site-kit' ) }
+			</p>
```

Comment:
The font sizes I'm seeing while testing here are a bit odd, the heading is just as small as the paragraph. This might be a result of my theme (Twenty Twenty-One), but we should probably try to use some more specific styles. Potentially that's worth a separate issue though, since it wouldn't be anything introduced by this PR (plus this one's already quite big).

Reply:
I've used the existing `googlesitekit-adminbar__title` and `googlesitekit-adminbar__subtitle` classes here to unify the size:

<img width="1231" alt="Screenshot 2021-01-26 at 15 06 58" src="https://user-images.githubusercontent.com/119352/105862856-2bfd3d00-5fe8-11eb-85de-a5ba9820c003.png">